src/components/FavoritesBtn.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+
+import api from '../service/api';
+
+import '../styles/FavoritesBtn.scss';
+
+type FavoritesBtnProps = {
+  movieId: number;
+};
+
+const FavoritesBtn: React.FC<FavoritesBtnProps> = ({ movieId }) => {
+  const [isFavorite, setisFavorite] = useState(api.isIdInFavorites(movieId));
```

Comment:
Please check the name of setisFavorite - starting from the second, each word should start with a capital letter

Reply:
Thanks for a hint. I have not noticed it at the first place